elasticsearch_collectd.py

```diff
@@ -572,6 +915,21 @@ def init_stats():
     if ES_VERSION.startswith("2."):
         NODE_STATS_CUR.update(NODE_STATS_ES_2)
 
+    # Attempt to parse the major, minor, and revision
```

Comment:
It does except in the case of alpha's and betas.  The revision number will sometimes have a -alpha or -beta appended to it.  5.0.0-alpha will not work.